packages/netlify-cms-core/src/components/Collection/Collection.js

```diff
@@ -29,10 +29,8 @@ class Collection extends React.Component {
   };
 
   renderEntriesCollection = () => {
-    const { name, collection } = this.props;
-    return (
-      <EntriesCollection collection={collection} name={name} viewStyle={this.state.viewStyle} />
-    );
+    const { collection } = this.props;
+    return <EntriesCollection collection={collection} viewStyle={this.state.viewStyle} />;
```

Comment:
`name` is not actually passed in to `Collection`, neither is it used by `EntriesCollection`. (Unless it is some "special" key that doesn't actually get passed though, we can remove it.)